build.sbt

```diff
@@ -29,6 +30,13 @@ lazy val anorm = project
       sourceManaged in Compile).map(m => Seq(GFA(m / "anorm"))),
     scalacOptions += "-Xlog-free-terms",
     binaryIssueFilters ++= Seq(
+      ProblemFilters.exclude[MissingClassProblem]("anorm.MayErr"),
+      ProblemFilters.exclude[MissingClassProblem]("anorm.MayErr$"),
+      // was deprecated:
```

Comment:
There are lots of miscellaneous implementation changes in anorm that change compatibility slightly. I know release compatibility is boring, but it's something we worry about. :)

Maybe we could bump the anorm version so that users can tell that there have been changes? Now that anorm is separated out from Play we can do that much more easily.